### PR TITLE
Fibonaci solved by using vectors

### PR DESCRIPTION
**Pull request by gitbart74859**

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: fibonacci
🏅 Granted 6 XP for @gitbart74859

File: homework/fibonacci/fibonacci.hpp [hunk 1]
```diff
@@ -1,11 +1,23 @@
...
 
-int fibonacci_recursive(int sequence) {
-    // TODO: Your implementation goes here
-    return 0;
+    return fibonacciVec[sequence];
 }
+
+int fibonacci_recursive(int n) 
+{
+    if(n <= 1)
```
Comment (nauka-programowania-MA) [inline homework/fibonacci/fibonacci.hpp:20]: space

File: homework/fibonacci/fibonacci.hpp [hunk 2]
```diff
@@ -1,11 +1,23 @@
...
-    // TODO: Your implementation goes here
-    return 0;
-}
+int fibonacci_iterative(int sequence) 
+{
+    std::vector<int>fibonacciVec {0 , 1};
+    fibonacciVec.reserve(sequence);
+
+    for(int i = 2; i <= sequence; i++)
+    {
```
Comment (nauka-programowania-MA) [inline homework/fibonacci/fibonacci.hpp:10]: space

File: homework/fibonacci/fibonacci.hpp [hunk 3]
```diff
@@ -1,11 +1,23 @@
 #pragma once
+#include <vector>
 
-int fibonacci_iterative(int sequence) {
-    // TODO: Your implementation goes here
-    return 0;
-}
+int fibonacci_iterative(int sequence) 
+{
+    std::vector<int>fibonacciVec {0 , 1};
```
Comment (nauka-programowania-MA) [inline homework/fibonacci/fibonacci.hpp:6]: space

**Review by nauka-programowania-MA — APPROVED** [review]

**Pull request closed without merging**